x-pack/packages/kbn-elastic-assistant-common/impl/llm/actions_client_llm.ts

```diff
@@ -52,8 +54,11 @@ export class ActionsClientLlm extends LLM {
     model,
     request,
     temperature,
+    traceOptions,
   }: ActionsClientLlmParams) {
-    super({});
+    super({
+      callbacks: [...(traceOptions?.tracers ?? [])],
```

Comment:
Just noting from our pairing discussions yesterday that we wanted to include the `APMTracer()` here as well, but since it lives in the plugin (and `ActionsClientLLM` has been moved to the `common` package), we need to do a little refactoring first before adding something like:

https://github.com/elastic/kibana/blob/56fe25ddc7bae4318cb20e13eb855e73b337f140/x-pack/plugins/elastic_assistant/server/lib/langchain/executors/openai_functions_executor.ts#L101